internal/docker/container.go

```diff
@@ -25,31 +25,30 @@ type ContainerRunner struct {
 	containerConfig *containerConfig
 }
 
-func (r *ContainerRunner) pullImage(img config.Image) error {
+func (r *ContainerRunner) pullImage(img string) error {
 	// Check docker image name property from the config file.
-	if img.Name == "" {
+	if img == "" {
 		return errors.New("no docker image specified")
 	}
 
 	// Check if image exists.
-	baseImage := r.docker.GetImageFlavor(img)
-	hasImage, err := r.docker.HasBaseImage(r.Ctx, baseImage)
+	hasImage, err := r.docker.HasBaseImage(r.Ctx, img)
 	if err != nil {
 		return err
 	}
 
 	// If it's our image, warn the user to not use the latest tag.
-	if strings.Index(img.Name, "saucelabs") == 0 && img.Tag == "latest" {
+	if strings.Index(img, "saucelabs") == 0 && strings.Contains(img, "latest") {
```

Comment:
Maybe it should be checked that it is ending by `:latest` ?

Reply:
aye, that'd be better 👍